manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+PackageLocale: en-US
+Publisher: Zimwiki
+PackageName: Zim

```

```suggestion
PackageName: Zim Desktop Wiki
```
This must match Control Panel.